backend/src/routes/v0/llm.py

```
@@ -203,11 +204,28 @@ async def optimize_prompt(
     operation_id="ruska_list_models",
     tags=["mcp"],
 )
-async def list_models():
+async def list_models(
+    user: ProtectedUser = Depends(get_optional_user_from_token),
```

Ruff (0.14.14)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)